pkg/api/v1/server_list_params.go

```diff
@@ -12,7 +13,8 @@ import (
 
 // ServerListParams allows you to filter the results
 type ServerListParams struct {
-	FacilityCode                 string `form:"facility-code"`
+	FacilityCode                 string    `form:"facility-code"`
+	DeletedAt                    time.Time `form:"deleted-at"`
```

Comment:
This should be more like `IncludeDeleted` and be a bool. Then the `setQuery` and `queryMods` below need to be updated to set/use this value.